core/controllers/admin.py

```diff
@@ -848,3 +848,21 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class InteractionsByExplorationId(base.BaseHandler):
+    """Handler to retrive the list of interaction id's for a exploration id."""
+
+    @acl_decorators.can_access_admin_page
+    def get(self):
+        exploration_id = self.request.get('exploration_id')
+        exploration = exp_fetchers.get_exploration_by_id(
+            exploration_id, strict=False
+        )
+        if exploration is None:
+            raise Exception('Exloartion does not exist.')
+        else:
```

Comment:
We don't need to specify `else` condition here, right?

Reply:
Oh yeah correct

Reply:
@aggarwalpulkit596, is this comment resolved?

Reply:
yes done